### DISCUSSION TITLE
Exotic keyboard layouts don't work

### DISCUSSION THREAD

**Discussion by KorcakDaniel** [hyprwm/Hyprland#5313] [Q&A] [answered]
Hi, I've wanted to include Neo2 Bone into my Hyprland config, but it seems that it doesn't work. Instead it switches to the default US layout.

The keyboard layout is included in the kdb upstream as well as listed in the locale, and works outside of the Hyprland.

`localectl list-keymaps | grep -i bone` prints the layout. It's not in the `/usr/share/X11/xkb/rules/base.lst` however. 
The original Neo is, but that doesn't work either. Any clues? 

I would very much like it to work.
Thank you.

**Comment by vaxerski** [comment]
that's up to xkb iirc, not hyprland

**Comment by KorcakDaniel** [comment, marked as answer]
Thank you, after a bit of digging I found that it's indeed present on the system as a variant under the DE layout, even though xkb didn't list it.

For those interested:
```
input {
    kb_layout = de
    kb_variant = bone
}
```
